apps/src/code-studio/pd/workshop_survey/VariableFormGroup.jsx

```diff
@@ -205,16 +206,32 @@ const VariableFormGroup = React.createClass({
       return (<FormGroup />);
     }
 
-    const columnQuestions = this.props.columnVariableQuestions.map(question => (
-      <ColumnVariableQuestion
-        key={question.name}
-        question={question}
-        selectedValues={this.state.selected}
-        data={this.props.data}
-        errors={this.props.errors}
-        onChange={this.props.onChange}
-      />
-    ));
+    let columnQuestions;
+    if (this.state.selected.length < 1) {
```

Comment:
What happens if there are no facilitators to choose from? Do we assume there will always be at least one facilitator?